.github/workflows/kcp-image.yaml

```diff
@@ -42,3 +43,17 @@ jobs:
         registry: ghcr.io/${{ github.repository_owner }}
         username: ${{ github.actor }}
         password: ${{ github.token }}
+
+    - name: Set up kubectl and kustomize
+      run: |-
+        curl -sL "https://github.com/kubernetes-sigs/kustomize/releases/download/kustomize%2Fv4.5.1/kustomize_v4.5.1_linux_amd64.tar.gz" | tar xzf -
```

Comment:
No way to just pre-install these? Hate to see big downloads per-run

Reply:
I looked at the kubectl github actions for a little bit, and they all either embedded the kubectl executable in a container image or they ran a container image that downloaded kubectl every time. So I figured I'd just cut out the middle man.

We could always create a custom runner that had all our binaries pre-installed.  This could also make caching image layers more straightforward.

Reply:
for the record: kubectl is 46mb.

Reply:
RH does maintain an [action](https://github.com/marketplace/actions/openshift-tools-installer ) that does proper caching it supports several tools would that help?

Reply:
How about we create a follow up issue to improve on this later?

Reply:
https://github.com/kcp-dev/kcp/issues/621